numpy/array_api/_creation_functions.py

```diff
@@ -148,6 +148,8 @@ def eye(
     _check_valid_dtype(dtype)
     if device not in ["cpu", None]:
         raise ValueError(f"Unsupported device {device!r}")
+    if k is None:
+        k = 0
```

Comment:
I could imagine that this  will require a bit more discussion:
The array API specifies that `None` is a legal value for `k` in the case of `eye`, but as it turns out this support is lacking from `np.eye` (which is called under the hood). 

``` python
In [4]: np.eye(3, k=None)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-0a5704552ae5> in <module>
----> 1 np.eye(3, k=None)

~\Documents\GitHub\numpy\numpy\lib\twodim_base.py in eye(N, M, k, dtype, order, like)
    213         M = N
    214     m = zeros((N, M), dtype=dtype, order=order)
--> 215     if k >= M:
    216         return m
    217     if k >= 0:

TypeError: '>=' not supported between instances of 'NoneType' and 'int'
```

Reply:
I'm guessing this is a typo in the spec, since the spec never mentions None other than by the use of `Optional`.

Reply:
Support for `k=None` has been removed from the spec as of https://github.com/data-apis/array-api/pull/278.